addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyMissingScanRule.java

```diff
@@ -146,4 +121,33 @@ private String getAlertAttribute(String key) {
     public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
+
+    private boolean hasCspHeader(HttpMessage msg) {
+        // Content-Security-Policy is supported by Chrome 25+, Firefox 23+, Safari 7+, but not but
+        // Internet Exploder
```

Comment:
Remove the comment, in which versions is was introduced it's not relevant to whether we should raise an alert or not (it's also not grammatically correct, "but not but", and Explorer has a typo).

Reply:
The explorer typo might have been introduced by me previously as a joke 😉